pyaedt/modeler/Primitives3D.py

```diff
@@ -45,6 +47,56 @@ def is3d(self):
 
         """
 
+    @aedt_exception_handler
+    def create_point(self, position, name=None, color="(143 175 143)"):
+        """Create a point.
+
+        Parameters
+        ----------
+        position : list
+            List of ``[x, y, z]`` coordinates. Note, The list can be empty or contain less than 3 elements.
+        name : str, optional
+            Name of the point. The default is ``None``, in which case the
+            default name is assigned.
+        color : str, optional
+            String exposing 3 int values such as "(value1 value2 value3)". Default value is ``"(143 175 143)"``.
+
+        Returns
+        -------
+        :class:`pyaedt.modeler.Object3d.Object3d`
```

Comment:
Are you sure it returns Object3d and not Point Object?

Reply:
@maxcapodi78 Great catch!